src/partners.html

```diff
@@ -0,0 +1,174 @@
+  <div class="clipped-header background-purple-4 col-11 col-md-10 mt-4 mt-md-5 mb-4 py-4 py-md-5 px-4 px-md-0">
+    <div class="offset-md-2 col-md-8 py-3 py-md-5 px-2 px-md-2">
+      <h1 class="text-white mb-3">Partners</h1>
+      <p class="text-white mb-3">
+        Bringing a modern and consistent transportation experience to all Californians requires help from a broad range of
+        experts. That's why Cal-ITP is proud to work collaboratively with our partners.
+      </p>
+    </div>
+  </div>
+</div>
+
+<section class="row justify-content-center">
+  <div class="col-md-10 col-11 px-0 mx-0">
+    <div class="offset-md-2 col-md-8 mt-md-4 mt-2">
+      <section class="pb-md-2">
+        <img class="w-100 rounded-1" src="/images/partners-cdt-background.png" />
+        <h2 class="my-md-4 my-3">California Department of Technology</h2>
+        <p>
+          We collaborate with the
+          <a class="link-body-emphasis fw-normal" href="https://cdt.ca.gov/" target="_blank"
+            >California Department of Technology</a
+          >
+          (CDT) to provide transit riders with a seamless way to qualify for and enroll in transit benefits online.
+        </p>
+        <figure class="quote my-md-4 my-3">
+          <blockquote>
+            “The Cal-ITP project offered us a test-case to deliver dependable and equitable public services, and reduce outdated
+            paper-based processes.”
+          </blockquote>
+          <figcaption>- Greg Fair, California Digital Identity Chief and Acting Chief Product Officer at CDT</figcaption>
+        </figure>
+        <p class="mb-0">Together, we aim to:</p>
+        <ul>
+          <li>
+            Expand
+            <a target="_blank" class="link-body-emphasis fw-normal" href="https://benefits.calitp.org/">Cal-ITP Benefits</a>, our
+            web application that uses Login.gov and the
+            <a
+              href="https://digitalidstrategy.cdt.ca.gov/primary-elements.html"
+              target="_blank"
+              class="link-body-emphasis fw-normal"
+              >California Identity Gateway</a
+            >
+            to safely and securely provide digital eligibility checks for transit benefits.
+            <ul>
+              <li>Transit riders who are older adults and U.S. Veterans can use the service today.</li>
+              <li>
+                Support eligibility checks for CalFresh cardholders, Medicare cardholders, and people with disabilities in the
+                coming year.
+              </li>
+            </ul>
+          </li>
+          <li>Promote the use of a Digital ID system that can be used across state departments. </li>
+          <li>
+            Demonstrate how we can deliver dependable and equitable public services while reducing the need for outdated
+            paper-based processes. 
+          </li>
+        </ul>
+        <div class="row justify-content-md-center align-content-stretch my-md-4 my-3">
```

Comment:
Same issue here with too much vertical spacing, since the `<ul>` has a bottom margin of 16px already.